packages/ringcentral-integration/lib/callLogHelpers.js

```diff
@@ -23,6 +23,9 @@ export function isOutbound(call = {}) {
 export function isRinging(call = {}) {
   return call.telephonyStatus === telephonyStatuses.ringing;
 }
+export function isRingingInboundCall(call = {}) {
```

Comment:
why need default value?  delegate the job to the isRinging()& isInbound(), since what you really do is a combination

Reply:
to prove what I said, I will use the type validation here:
since the signature of the `isRing` and `isIncoming` are both the same signature *F*:
```
obj->bool
```
and the `&&` operator can be seen as the function with signature like:
```
bool->bool->bool
```
so to get what you want by using the `isRing` and `isIncoming` , you need:
```
F(obj)->F(obj)->bool
```
which tells you that you need a *functor*:
```javascript
const boolFunctor = function (bool){
  return function(func1){
    return function(func2){
      return function(val){
        return bool(func1(val), func2(val))
      }
    }
  }
}
// suppose *and* is a function too
boolFunctor(and)(isIncomingCall)(isRingCall)
```
which is semantically equal to what you just wrote.
And in the code above, you can see there is no sign of the value to apply, because you don't need it.

And since we are writing the javascript, the implementation above does not necessarily needed, even thought, this give the mathematical proof of what I mean